api/aperture/cloud/cmd/v1/cmd.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+// Messages for aperturectl → cloud controller communication.
+package aperture.cloud.cmd.v1;
+
+import "aperture/policy/language/v1/policy.proto";
+import "google/protobuf/empty.proto";
+
+// CloudController describes APIs of the cloud controller from the aperturectl POV
+service CloudController {
```

Comment:
Again – I regret making a big `service Controller` which has lot of different stuff. Since we're starting from scratch here, can we do fine-grained services from the start?

I'd call it `PolicyService` or so.

Reply:
Made the changes